ui/appui-react/src/appui-react/widgets/WidgetDef.tsx

```diff
@@ -543,10 +551,93 @@ export class WidgetDef {
     if (!state) return;
     if (!frontstageDef.findWidgetDef(this.id)) return;
 
-    frontstageDef.dispatch({
-      type: "WIDGET_TAB_SHOW",
-      id: this.id,
-    });
+    const tabLocation = getTabLocation(state, this.id);
+    if (tabLocation && isPopoutTabLocation(tabLocation)) {
+      const testWindow = UiFramework.childWindows.find(
+        tabLocation.popoutWidgetId
+      );
+      if (testWindow) {
+        // We have to check if Safari is the browser being used. Safari has security
+        // measures that do not allow you to change focus to another tab programmatically.
+        // Therefore the only and recommended way around this at the moment, is to close
+        // and then open the tab again. This remounts the popout. Thats why this will not be done,
+        // because we don't want the state of the widget to be lost. Instead a toast will
+        // be displayed that informs the user that safari cannot focus the popout.
+        // Apple support docs: https://discussions.apple.com/thread/251676767?sortBy=best
+        const isSafari =
+          navigator.userAgent.toLowerCase().indexOf("safari/") > -1;
+
+        const isChrome =
+          navigator.userAgent.toLowerCase().indexOf("chrome") > -1;
+
+        if (!isChrome && isSafari) {
```

Comment:
@FlyersPh9 I have decided to remove the toast message for now, since we are only checking one potential case of failure, but it seems like there could be multiple other cases and there's no good way to check if the `focus` had any effect.